### ISSUE TITLE
Combined-method survey effort evaluation

### ISSUE THREAD

**Issue by mkclapp** [calacademy-research/COMB#30]
[low-ish priority but would be highly useful to investigate prior to field season 2022]

GOAL: to evaluate the impact of different levels of survey effort (# of in-person point count surveys) on the quality of the occupancy model that uses both ML and point count data.

Using the simulation described in AHM2 7.6.1, simulate different numbers of visits in the y matrix and evaluate effects on occupancy model (see p 428).

More complicated applications (using our real data, multiple species, etc) can follow...

**Comment by mschulist** [comment]
This is something that I really want to work on once we have the models completed. Super cool!

**Comment by mkclapp** [comment]
> This is something that I really want to work on once we have the models completed. Super cool!

Awesome, Mark! Looking forward to it! We are making good progress this week, so maybe quite soon..

**Comment by mkclapp** [comment]
Bookmarking AHM1 Ch 10 pp.584-6, and AHM2 Ch 7 pp.428-9 for some guidance on this

**Comment by ddkapan** [comment]
- [ ] Still interesting @mkclapp @mschulist ?

**Comment by mschulist** [comment]
> * [ ]  Still interesting @mkclapp @mschulist ?

Yeah, especially once the model is working to some extent. I can write a script that will feed the model differing amounts of data and can compare how the model does (confidence?) with the different amounts of data.

**Comment by mkclapp** [comment]
> > * [ ]   Still interesting @mkclapp @mschulist ?
> 
> Yeah, especially once the model is working to some extent. I can write a script that will feed the model differing amounts of data and can compare how the model does (confidence?) with the different amounts of data.

+1! great Mark! We will let you know as soon as it is...
